src/paperoni/__main__.py

```diff
@@ -163,21 +169,51 @@ class View:
         # Output format
         format: Formatter = TerminalFormatter
 
+        # Number of papers to view
+        n: int = None
+
+        def run(self, work):
+            it = itertools.islice(work.top, self.n) if self.n else work.top
+            self.format(Scored(ws.score, ws.value.current) for ws in it)
```

Comment:
It's a bit late now to ask that question but I'm wondering why is `PaperWorkingSet` a collection or why do we have multiple `PaperWorkingSet` of len 1?